uiowa/uiowa
# Iowa now contact

Resolves https://github.com/uiowa/uiowa/issues/5675
https://github.com/uiowa/uiowa/issues/5741

## How to test
- Sync now.uiowa.edu
- your config should import
- You should be able to add a contact, either a person or a contact, to a person or an article
- When you add a new person from the IEF, you should see a name in the table
- yeehaw
